cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,216 @@
-/// <reference types='cypress' />
+/// <reference types="cypress" />
+const { generateUser } = require('../support/generate');
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  const userName = 'test';
+  const checkEmail = [
+    'cierra@example.com',
+    'alden@example.com',
+    'kierra@example.com'
+  ];
+  const {
+
+    lastNameFaker,
+    emailFaker,
+    ageFaker,
+    salaryFaker,
+    departmentFaker,
+    name,
+    newAge,
+    Email,
+    LastName,
+    Age,
+    Salary,
+    Department
+
+  } = generateUser();
+
+  it('should provide an ability to pagination', () => {
+    cy.get('.-pageJump > input')
+      .should('have.value', '1');
+  });
+
+  it('should provide an ability to rows count selection', () => {
+    cy.get('[aria-label="rows per page"]')
+      .select('5 rows');
+
+    cy.get('[value="5"]')
+      .should('have.value', '5');
+
+    cy.get('select')
+      .should('have.value', '5');
+
+    cy.get('[aria-label="rows per page"]')
+      .select('10 rows');
+
+    cy.get('[value="10"]')
+      .should('have.value', '10');
+
+    cy.get('select')
+      .should('have.value', '10');
+
+    cy.get('[aria-label="rows per page"]')
+      .select('20 rows');
+
+    cy.get('[value="20"]')
+      .should('have.value', '20');
+
+    cy.get('select')
+      .should('have.value', '20');
+
+    cy.get('[aria-label="rows per page"]')
+      .select('25 rows');
+
+    cy.get('[value="25"]')
+      .should('have.value', '25');
+
+    cy.get('select')
+      .should('have.value', '25');
+
+    cy.get('[aria-label="rows per page"]')
+      .select('50 rows');
+
+    cy.get('[value="50"]')
+      .should('have.value', '50');
+
+    cy.get('select')
+      .should('have.value', '50');
+
+    cy.get('[aria-label="rows per page"]')
+      .select('100 rows');
+
+    cy.get('[value="100"]')
+      .should('have.value', '100');
+
+    cy.get('select')
+      .should('have.value', '100');
+  });
+
+  it('should provide an ability to add new worker', () => {
+    cy.contains('.btn', 'Add')
+      .click();
+    cy.get('[id="firstName"]')
+      .type(userName);
+    cy.get('[id="lastName"]')
+      .type(lastNameFaker);
+    cy.get('[id="userEmail"]')
+      .type(emailFaker);
+    cy.get('[id="age"]')
+      .type(ageFaker);
+    cy.get('[id="salary"]')
+      .type(salaryFaker);
+    cy.get('[id="department"]')
+      .type(departmentFaker);
+    cy.contains('.btn', 'Submit')
+      .click();
+    cy.get('[id="firstName"]')
+      .should('contain', userName);
+    cy.get('[id="lastName"]')
+      .should('contain', lastNameFaker);
+    cy.get('[id="userEmail"]')
+      .should('contain', emailFaker);
+    cy.get('[id="age"]')
+      .should('contain', ageFaker);
+    cy.get('[id="salary"]')
+      .should('contain', salaryFaker);
+    cy.get('[id="department"]')
+      .should('contain', departmentFaker);
```

Comment:
Looks like you are asserting the same selectors as you were filling in. Is it expected?